action.yml

```diff
@@ -26,6 +26,12 @@ inputs:
   githubToken:
     description: 'The Github Token to be used to authenticate with Vault'
     required: false
+  roleName:
```

Comment:
A little confused if this is the name of the K8s service account or the Vault role?

If it's the service account maybe `serviceAccountName` is a better parameter name?

Reply:
I can do that